lmcache/cache_engine.py

```diff
@@ -380,6 +406,49 @@ def retrieve(
 
         return ret, ret_mask
 
+    @_lmcache_nvtx_annotate
+    @torch.no_grad()
+    def lookup(
+        self,
+        tokens: torch.Tensor,
+        align_to_chunk: bool,
```

Comment:
When will `align_to_chunk = True` be used?

Reply:
When skipping some tokens in store(used together with kv_tensor_mask in store), it should skip a chunk multiple tokens. Here aligh_to_chunk should be True.